src/gc/gcinterface.h

```diff
@@ -727,6 +731,7 @@ class IGCHeap {
     SVAL_DECL(uint32_t, gcHeapType);
 #endif
 
+    static const uint32_t minFinalizingGeneration;
```

Comment:
Ideally there won't be any static fields on `IGCHeap` - the two existing ones (`gcHeapType` and `maxGeneration`) will be getting removed soon. Can we avoid putting this on the interface itself? (Provided it's necessary, per Jan's comment)